src/InfluxDB/Point.php

```diff
@@ -81,26 +88,39 @@ public function __toString()
         return $string;
     }
 
+    /**
+     * @param  array $arr
+     * @return string
+     */
     private function arrayToString(array $arr)
     {
         $strParts = array();
 
         foreach ($arr as $key => $value) {
-            $strParts[] = "{$key}={$value}";
+            $strParts[] = sprintf('%s=%s', $key, $value);
```

Comment:
about https://github.com/influxdb/influxdb-php/blob/fd80291e151c4eba966f85165082fc920c111670/src/InfluxDB/Client.php#L143

I saw it and then i both giggled and cried a little bit.